handbook/engineering/README.md

```diff
@@ -250,6 +252,18 @@ Our goal is to keep these dependencies up-to-date with each release of Fleet. If
 
 Next, create a new GitHub issue using the [Release QA template](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=&projects=&template=smoke-tests.md&title=). Add the release version to the title, and assign the quality assurance members of the [MDM](https://fleetdm.com/handbook/company/development-groups#mdm-group) and [CX](https://fleetdm.com/handbook/company/development-groups#customer-experience-group) product groups.
 
+### Merging during the freeze period
+
+We merge bug fixes during the freeze period, but we do not merge other code changes. This minimizes code churn and helps ensure a stable release. To merge a bug fix, you must first unfreeze the PR in [Merge Freeze](https://app.mergefreeze.com/installations/3704/branches), and click the "Unfreeze 1 pull request" text link. 
```

Comment:
```suggestion
We merge bug fixes and documentation changes during the freeze period, but we do not merge other code changes. This minimizes code churn and helps ensure a stable release. To merge a bug fix, you must first unfreeze the PR in [Merge Freeze](https://app.mergefreeze.com/installations/3704/branches), and click the "Unfreeze 1 pull request" text link. 
```

Small suggestion to add docs here^. 

Then, I think we can get rid of this note below to dedup (note is in the [Freeze day section](https://fleetdm.com/handbook/engineering#freeze-day)):

![Screenshot 2023-08-31 at 2 57 16 PM](https://github.com/fleetdm/fleet/assets/47070608/380579cf-6011-4ecd-960c-5b9084f9be65)